docs/en/01-how-to-build/cmake_option.md

```diff
@@ -0,0 +1,80 @@
+# CMake Build Option Spec
+
+<table class="docutils">
+<thead>
```

Comment:
This document should update more options, such as `MMDEPLOY_BUILD_EXAMPLES`, `MMDEPLOY_BUILD_SDK_MONOLITHIC`, `MMDEPLOY_COVERAGE`, and so on.